qcoverage/test/org/qcmg/coverage/CoverageTest.java

```diff
@@ -0,0 +1,161 @@
+package org.qcmg.coverage;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.qcmg.common.commandline.Executor;
+import org.qcmg.qio.gff3.Gff3Record;
+import org.qcmg.qio.record.RecordWriter;
+
+import htsjdk.samtools.SAMFileHeader.SortOrder;
+
+public class CoverageTest {
+	private String inputBam;
+	private String inputBai;
+	private String inputGff3;
+	private String fname;
+	private String log;
+
+	
+	 @Rule
+	 public TemporaryFolder testFolder = new TemporaryFolder();
+	
+	
+	@Before
+	 public void setup() throws IOException {
+		 inputBam = testFolder.newFile("coverage.bam").getAbsolutePath();
+		 inputBai = inputBam.replace("bam", "bai");
+		 SequenceCoverageTest.createCoverageBam(inputBam, SequenceCoverageTest.getAACSAMRecords(SortOrder.coordinate), SequenceCoverageTest.createSamHeaderObject(SortOrder.coordinate));
+		 
+			Gff3Record record = new Gff3Record();
+			record.setSeqId("chr1");
+			record.setType("exon");
+			record.setStart(100);
+			record.setEnd(102);
+			record.setScore(".");
+			record.setSource(".");
+			record.setStrand("+");
+		
+		 
+		File file = testFolder.newFile("test.gff3");  
+		try (RecordWriter<Gff3Record> writer = new RecordWriter<>(file)) {
+			writer.add(record);
+		}
+		inputGff3 = file.getAbsolutePath();
+		log = inputBam.replace("bam", "log");
+		fname = testFolder.getRoot().getAbsolutePath()+"/output";
+	 } 
+	
+	@Test
+	public final void DefaultTest() throws Exception {
```

Comment:
doesn't follow Java's naming convention for methods - pls check all methods for this!

Reply:
fixed